lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -93,7 +93,7 @@ def fetch_pod_metadata(namespace_name, pod_name)
             log.trace("returning empty metadata for #{namespace_name}/#{pod_name} due to error") if log.trace?
           end
         end
-      rescue KubeException=>e
+      rescue Exception=>e
         @stats.bump(:pod_cache_api_nil_error)
         log.debug "Exception encountered fetching pod metadata from Kubernetes API #{@apiVersion} endpoint #{@kubernetes_url}: #{e.message}"
```

Comment:
`"Exception #{e} encountered ....` - when debugging, it is good to know exactly what the exception is - same below

Reply:
Updated